README.md

```diff
@@ -1 +1,45 @@
-# operator-workflows
\ No newline at end of file
+# operator-workflows
+
+## Description
+
+This repository includes several parametrized CI workflows to be used for both kubernetes and machine charms.
+
+## Usage
+
+To use the workflows in your repository, just reuse them including the `secrets: inherit` key. Examples of usage for every workflow can be found at the (indico operator)[https://github.com/canonical/indico-operator/blob/main/.github/workflows].
+
+The following workflows are available:
+
+* test: executes the default tox targets defined in the `tox.ini`file and generates a plain text report. The following parameters are available for this workflow:
+
+| Name | Type | Default | Description
+| runs-on| string | ubuntu-20.04 | Image runner for the test execution |
+
+* comment: Posts the content of the artifact specified as a comment in a PR. It needs to be triggered from a PR triggered workflow.
+
+* integration_test: Builds the existing Dockerfiles, if any, and executes the integration test target defined in the `tox.ini` file. The following parameters are available for this workflow:
```

Comment:
LGTM, but do you think it would be advisable to keep an eye on readme's line length?

Reply:
It seems to fit the diff window :)